app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt

```diff
@@ -1,32 +1,72 @@
 package com.lightricks.feedexercise.ui.feed
 
+import android.content.Context
 import androidx.lifecycle.*
+import androidx.room.Room
 import com.lightricks.feedexercise.data.FeedItem
+import com.lightricks.feedexercise.data.FeedRepository
+import com.lightricks.feedexercise.database.FeedDatabase
+import com.lightricks.feedexercise.network.FeedApiService
 import com.lightricks.feedexercise.util.Event
-import java.lang.IllegalArgumentException
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+
 
 /**
  * This view model manages the data for [FeedFragment].
  */
-open class FeedViewModel : ViewModel() {
+open class FeedViewModel(service: FeedApiService, private val db: FeedDatabase) : ViewModel() {
```

Comment:
Injecting one `FeedRepository` instead of two things (service and the database) would be even better.